### ISSUE TITLE
I just get the acc of 0.9% in your dataset when I use PVT. It's too lower that I can't believe, 

### ISSUE THREAD

**Issue by 1104662797** [hendrycks/natural-adv-examples#11]
I just use the code of `https://www.kaggle.com/paultimothymooney/starter-kernel-for-imagenet-a-adversarial-examples` and load the model in the follow way`net = pvt_medium()
checkpoint = torch.load("../input/pvt-medium/4f268100-d9ba-11eb-8129-547eacb8081d", map_location='cpu')
if 'model' in checkpoint:
    msg = net.load_state_dict(checkpoint['model'])
else:
    msg = net.load_state_dict(checkpoint)
net.cuda()
net.eval() `  the code and model of pvt is there`https://github.com/whai362/PVT/tree/v2/classification`.
 I get the result `
pvt_medium Accuracy on ImageNet-A Dataset (%): 0.9333
				Baseline
RMS Calib Error (%): 		51.24
AURRA (%): 			0.96
` As you can see, the result is very bad.
Is there anything wrong? could you help me to fix my mistake. Thanks very much.

**Comment by 1104662797** [comment]
The newest code resolve it.
